### ISSUE TITLE
Hours label and input don't match

### ISSUE THREAD

**Issue by maya** [18F/tock#531]
For is `hours` but the id is `id_timecardobject_set-0-hours_spent` and this id changes for each input.  They should be the same so they're explicitly associated with each other per W3C F68 4.1.2 https://www.w3.org/TR/WCAG20-TECHS/F68.

```
<div class="entry-amount">
  <label for="hours">Hours Spent:</
  <input id="id_timecardobject_set-0-hours_spent" min="0" name="timecardobject_set-0-hours_spent" step="0.25" type="number">
</div>
```

**Comment by maya** [comment]
Closing in favor of consolidated issue: #533.
